include/util/query_heap.hpp

```diff
@@ -356,13 +356,13 @@ class QueryHeap
         const auto index = node_index.peek_index(node);
         auto &reference = inserted_nodes[index];
         reference.weight = weight;
-        heap.increase(reference.handle, std::make_pair(weight, index));
+        heap.decrease(reference.handle, std::make_pair(weight, index));
```

Comment:
This change is pretty fundamental to how the whole routing algorithm works.  Can you explain exactly why this change is being made here?